src/Okta.Sdk/Client/Configuration.cs

```diff
@@ -758,7 +758,7 @@ public static string ToDebugReport()
             report += "    OS: " + System.Environment.OSVersion + "\n";
             report += "    .NET Framework Version: " + System.Environment.Version  + "\n";
             report += "    Version of the API: 5.1.0\n";
-            report += "    SDK Package Version: 7.0.2\n";
+            report += "    SDK Package Version: 7.0.3\n";
```

Comment:
Should we consider using the constant here?  Just to limit the number of places the version needs to be updated.

Reply:
This is autogenerated. It takes the version from the config file.